ui/components/data-sources/DataSourcesListItem.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {
+    ListView,
+    DropdownKebab,
+    MenuItem
+} from "patternfly-react";
+
+const DataSourcesListItem = ({ item }) => {
+    const { type, name } = item;
+
+    return (
+        <ListView.Item
+            className="ds-list-item"
+            heading={type}
+            description="---"
+            leftContent={(
+                <span className="list-item-name">{name}</span>
+            )}
+            actions={(
+                <div>
+                    <DropdownKebab id="data-source-list-item-dropdown" pullRight>
```

Comment:
You can already add the real options: `Edit Data Source` and `Delete Data Source`